src/index.js

```diff
@@ -0,0 +1,54 @@
+'use-strict'
+
+const pretty = require('pino-pretty')
+const dateformat = require('dateformat')
```

Comment:
`dateformat` is from the dependencies of `pino-pretty`. Do we wanna add `dateformat` as a dependency in this project?

Reply:
> Do we wanna add dateformat as a dependency in this project?

Yes, we should declare every dependency without relying on transitive ones (npm will dedupe them)